packages/core/src/types/oas3.ts

```diff
@@ -136,6 +136,7 @@ const Parameter: NodeType = {
     content: 'MediaTypeMap',
   },
   required: ['name', 'in'],
+  requiredOneOf: ['schema', 'content']
```

Comment:
would it work with OAS 3.1?

Reply:
Yes, because Oas3_1Types include Oas3Types (and Parameter type too):

Reply:
Do we have a code style for a trailing comma on these? I notice, for example, a trailing comma on the line above.

Reply:
Not sure.

Reply:
@adamaltman can I merge this or hold it till we found a common ground on commas?

Reply:
We do have but looks like this project does not have prettier set up. @tatomyr would be great to do it on the cool-down.

Reply:
@AntonKozachuk I added the comma 😄